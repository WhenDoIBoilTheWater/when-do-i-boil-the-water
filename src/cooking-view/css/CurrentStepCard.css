.current-step-card {
	background-color: var(--light-grey);
	border-radius: var(--border-radius);
	color: var(--dark-grey);
	display: flex;
	flex-direction: column;
	justify-content: space-between;

	max-width: 350px;
	min-width: 150px;

}
	.current-step-card h1 {
		margin-bottom: .5rem;
		padding: 1rem;
	}