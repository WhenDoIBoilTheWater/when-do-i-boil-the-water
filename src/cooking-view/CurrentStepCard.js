import React from 'react'
import BarTimer from './BarTimer.js'
import './css/CurrentStepCard.css'

class CurrentStepCard extends React.Component {
    // constructor(props){
    //     super(props)
    // }

    render(){
        return(
            <div className="current-step-card">
                <p>{this.props.recipe}:</p>
                <h1>{this.props.description}</h1>
                <BarTimer length={this.props.length} />
            </div>
        )
    }
}

export default CurrentStepCard