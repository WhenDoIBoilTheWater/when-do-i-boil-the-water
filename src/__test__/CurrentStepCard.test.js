import React from 'react';
import CurrentStepCard from '../cooking-view/CurrentStepCard.js';
import renderer from 'react-test-renderer';

it('renders NOTHING', () => {
  const tree = renderer.create().toJSON();
  expect(tree).toMatchSnapshot();
});
