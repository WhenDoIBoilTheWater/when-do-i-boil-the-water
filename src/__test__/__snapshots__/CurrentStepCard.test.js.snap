// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NOTHING 1`] = `null`;
